src/mvs_eland/C1_verification.py

```diff
@@ -166,7 +166,7 @@ def check_non_dispatchable_source_time_series(dict_values):
     """
     # go through all non-dispatchable sources
     for key, source in dict_values[ENERGY_PRODUCTION].items():
-        if source[RENEWABLE_ASSET_BOOL][VALUE] == True:
+        if RENEWABLE_ASSET_BOOL in source and source[RENEWABLE_ASSET_BOOL][VALUE] == True:
```

Comment:
Why not check `if TIMESERIES_NORMALIZED in source`?

Reply:
Good point, I'll change that in #498 as it is part of that PR - I thought here changes are only shown between the two branches I compare in this PR but seems like changes to the dev are shown.

Reply:
done